enzy_htp/chemical/residue.py

```diff
@@ -6,20 +6,217 @@
 """
 
 from typing import List, Set, Dict, Union
-from .db import load_from_db
 from ..core import InvalidResidueCode, _LOGGER
 
-AA_LIST: List[str] = load_from_db("AA_LIST")
+AA_LIST: List[str] = [
+        "R",
+        "H",
+        "K",
+        "D",
+        "E",
+        "S",
+        "T",
+        "N",
+        "Q",
+        "C",
+        "G",
+        "P",
+        "A",
+        "V",
+        "I",
+        "L",
+        "M",
+        "F",
+        "Y",
+        "W",
+        "U"
+]
 """Capitalized list of all one-letter amino acid names."""
 
-THREE_LETTER_AA_MAPPER: Dict[str, str] = load_from_db(
-    "THREE_LETTER_AA_MAPPER")  #@shaoqz: @imp2 add related to canonical in the name
+THREE_LETTER_AA_MAPPER: Dict[str, str] = {
+        "ALA": "A",
+        "ARG": "R",
+        "ASN": "N",
+        "ASP": "D",
+        "CYS": "C",
+        "GLN": "Q",
+        "GLU": "E",
+        "GLY": "G",
+        "HID": "H",
+        "HIE": "H",
+        "HIP": "H",
+        "HIS": "H",
+        "ILE": "I",
+        "LEU": "L",
+        "LYS": "K",
+        "MET": "M",
+        "PHE": "F",
+        "PRO": "P",
+        "SEC": "U",
+        "SER": "S",
+        "THR": "T",
+        "TRP": "W",
+        "TYR": "Y",
+        "VAL": "V"
+} #TODO(shaoqz): @imp2 add related to canonical in the name
```

Comment:
I added several alternative names for some residues. Please add it back

Reply:
Added extra codes back in.